mobile/lib/components/utility/shadowed_container.dart

```diff
@@ -31,12 +35,13 @@ class ShadowedContainer extends StatelessWidget {
       margin: margin,
       padding: padding,
       decoration: BoxDecoration(
-        color: shadowColor,
-        boxShadow: const [
+        color: backgroundColor,
+        shape: shape ?? BoxShape.rectangle,
+        boxShadow: [
           BoxShadow(
-            color: Colors.black38,
+            color: shadowColor ?? Colors.black38,
```

Comment:
shouldn't we use palette colors https://github.com/RadioAktywne/ra-app/blob/master/mobile/lib/resources/colors.dart?